sunpy/coordinates/tests/test_metaframes.py

```diff
@@ -127,7 +127,7 @@ def test_no_base():
 
 
 def test_no_obstime():
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="obstime"):
```

Comment:
This seems short? What was the full message?

Reply:
The base coordinate frame must have a defined `obstime`

Reply:
Then I think that should be added here in full.

Reply:
Done. Thank You.